db/db-migration.py

```diff
@@ -0,0 +1,9 @@
+from yoyo import read_migrations, get_backend
+from db_settings import postgresUser, postgresHost
```

Comment:
it's not db_settings but local_settings (there could be more settings in the future).

The mechanism go this way:
- you checkout the repo
- you copy local_settings_example.py to local_settings.py
- you edit local_settings.py
- you run

As told local_settings.py have to be in .gitignore.

The idea is that you give a template to maintainer, but the local_settings.py must be ignored to avoid leaking secrets by accident.
This is a well known pattern.

Reply:
Yep, I got it. Will integrate it into local_settings_example.py.